akka-actor-typed/src/main/scala/akka/actor/typed/delivery/internal/ConsumerControllerImpl.scala

```diff
@@ -274,7 +274,7 @@ private class ConsumerControllerImpl[A] private (
   import ProducerControllerImpl.Resend
   import settings.flowControlWindow
 
-  private val flightRecorder = ActorFlightRecorder.recorder
+  private val flightRecorder = ActorFlightRecorder
```

Comment:
And no need for this field or the parameter above, use the ActorFlightRecorder directly